Hangman/hangman.rb

```diff
@@ -0,0 +1,63 @@
+def contains?(character,key,current_word,guess_count,streak,score)
+  if key.include?(character)
+    i=0
+    if(current_word[key.index(character)]=="-")
+      while key[i..key.length].include?(character)
+        current_word[key[i..key.length].index(character)+i] = character
+        i = key[i..key.length].index(character)+i+1
+      end
+      streak += 1
+      score += streak*10
+    end
+    puts "Cool..."
+    return guess_count,score,streak
+  else
+    guess_count-=1
+    streak=0
+    puts "Oops!..."
+    return guess_count,score,streak
+  end
```

Comment:
as you are returning "return guess_count,score,streak" from both blocks you can take that out and write only once at then end of if/else